src/views/api/TryItOut.vue

```diff
@@ -129,7 +145,31 @@ const requests: Requests = {
         example: "10",
         tooltip: "How many results to return"
       }
-    ]
+    ],
+    tooltip: "How to search for gensets by keywords"
+  },
+  "Batch lookup": {
+    method: "POST",
+    path: "query/?",
+    fields: [
+      {
+        name: "q",
+        example: "WP661,WP4875,WP4918",
+        tooltip: "Comma or plus separated list of genesets to lookup"
+      },
+      {
+        name: "fields",
+        example: "genes.ensemblgene",
+        tooltip:
+          "Fields to return from each geneset. Accepts any Elasticsearch query syntax, e.g. * for wildcard."
+      },
+      {
+        name: "size",
+        example: "10",
+        tooltip: "How many results to return"
+      }
```

Comment:
This batch query doesn't need the  `size` parameter, as it only returns 3 results. A more useful field to include would be `scope=_id`